tests/Backend/Mixed/SharingTest.php

```diff
@@ -309,11 +328,11 @@ public function testSharedBuckets($backend)
         $this->initTestBuckets($backend);
         $bucketId = reset($this->_bucketIds);
 
-        $tableName = 'numbers';
+        $tableName = 'transactions';
```

Comment:
Zmenil som tabulku pretoze v hlavicke boli cisla a blbo sa s tim pracovalo, toto je tiez mala tabulka testy presli tak snad to nevadi

Reply:
LGTM